### PR TITLE
Feature wrap example

### PR DESCRIPTION
**Pull request by pmturner**

### CODE REVIEW

**Comment by shaedrich** [comment]
What's the difference to #971?

File: data/en/cfdefaultcase.json [hunk 1]
```diff
@@ -11,5 +11,19 @@
 		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/tag/cfdefaultcase"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/tag/cfdefaultcase"}
 	},
+	"examples": [
+		{
+			"title": "Script Syntax",
+			"description": "",
+			"code": "vegetable = \"\";\r\nswitch(vegetable) {\r\n    case \"carrot\":\r\n        writeOutput(\"Carrots are orange.\");\r\n        break;\r\n    default:\r\n        writeOutput(\"You don't have any vegetables!\");\r\n        break;\r\n}",
+			"result": "You don't have any vegetables!"
```
Comment (shaedrich) [inline data/en/cfdefaultcase.json]:
```suggestion
			"result": "You don't have any vegetables!",
			"runnable": true
```

**Comment by pmturner** [comment]
Kamasamak and Xcreen have it exactly right. I'm sorry about that. Would you prefer I close them and resubmit?

**Comment by shaedrich** [comment]
It would be best if you merge all into one and then close the others.

**Pull request closed without merging**